javascript/src/components/outputs/Output.jsx

```diff
@@ -29,51 +30,69 @@ var Output = React.createClass({
     _typeNotAvailable() {
         return (this.props.types[this.props.output.type] == undefined);
     },
+    getInitialState() {
+        return {};
+    },
+    componentDidMount() {
+        this.props.getTypeDefinition(this.props.output.type, (typeDefinition) => {
+            this.setState({typeDefinition: typeDefinition});
+        });
+    },
     /* jshint +W116 */
```

Comment:
The jshint comments should be for the `_typeNotAvailable` method. The first one disables the `==` check, and the second one enables it again for the rest of the file. Probably we should have placed the comments inside the method, but anyway :)